docs/kcl/map.md

```diff
@@ -45,7 +45,7 @@ circles = map([1..3], drawCircle)
 ```js
 r = 10 // radius
 // Call `map`, using an anonymous function instead of a named one.
-circles = map([1..3], (id) => {
+circles = map([1..3], (id) {
```

Comment:
How is this okay?

Reply:
Seems like the docs didn't get updated - there is a `fn` in the doc comment